src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -0,0 +1,130 @@
+1. Locate the subnet IDs corresponding to the VPC you chose in Step 4.
+1. Click **Security Groups** in the sidebar.
+1. <a name="step-8"></a> Click **Create security group** to create a security group within your VPC that allows inbound access from your EC2 instances on Port 26257:
+  - In the **Security group name** field, enter a name for the security group.
+  - In the **Description** field, enter a description for the security group.
+  - From the **VPC** dropdown, select the VPC you chose in Step 4.
+  - In the **Inbound rules** section, click **Add rule**. Enter *26257* in the **Port range** field. In the **Source** field, enter the CIDR range from Step 5.
+  - Click **Create security group**.
+
+Use either the Amazon VPC Console or the [AWS Command Line Interface (CLI)](https://aws.amazon.com/cli/) to continue:
+
+  <div class="filters clearfix">
+    <button style="width: 15%" class="filter-button" data-scope="aws-console">AWS Console</button>
+    <button style="width: 15%" class="filter-button" data-scope="aws-cli">AWS CLI</button>
+  </div>
+
+<section class="filter-content" markdown="1" data-scope="aws-console">
+
+1.  Click **Endpoints** in the sidebar.
+1.  Click **Create Endpoint**.
+1.  On the **Create Endpoint** page, for the **Service Category** field, select **Find service by name**.
+1.  In the **Service Name** field, enter the **Service Name** copied from the connection modal in [Step 1](#step-1).
+1.  Click **Verify**.
+1.  In the **VPC** field, enter the ID of the VPC you want to create your endpoint in.
+1.  Verify that the subnets are pre-populated.
+1.  In the **Security group** section, select the security group you created in [Step 8](#step-8) and uncheck the box for **default** security group.
+1.  Click **Create Endpoint**.
+
+      The VPC Endpoint ID displays.
+
+1.  Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+
+  </section>
+
+  <section class="filter-content" markdown="1" data-scope="aws-cli">
+
+1.  Substitute the values from the previous steps and run the following AWS CLI command:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ aws ec2 create-vpc-endpoint --region $REGION \
+    --vpc-id $VPC_ID --subnet-ids $SUBNET_ID1 $SUBNET_ID2 \
+    --vpc-endpoint-type Interface --security-group-ids \
+    $SECURITY_GROUP_ID1 $SECURITY_GROUP_ID2 --service-name \
+    $SERVICE_NAME_PROVIDED_BY_COCKROACH
+    ~~~
+
+1.  Locate the VPC Endpoint ID in the CLI output.
+
+1.  Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+
+  </section>
+
+### Verify the endpoint ID
+
+1.  Paste the Endpoint ID you created into the **VPC Endpoint ID** field.
+1.  Click **Verify**.
+1.  {{ site.data.products.db }} will accept the endpoint request. You can confirm the request acceptance by checking if the status is listed as Available on the Amazon VPC Console **Endpoints** page.
```

Comment:
Nit: "Available" --> "**Available**" ? (global), and perhaps the same for "Pending"